### PR TITLE
Update the frame rate limiter description

### PR DESCRIPTION
**Pull request by dora-0**
wiki/Options/en.md
wiki/Shortcut_Key_Reference/en.md

Changed from Stable 20191129.3

### CODE REVIEW

**Comment by bdach** [comment]
Might as well update the [*Options*](https://github.com/ppy/osu-wiki/blob/master/wiki/Options/en.md) article with the description of the new options while you're at it?

**Comment by dora-0** [comment]
@bdach I just updated it. Could you please check?

**Comment by bdach** [comment]
The additions seem fine to me; it also might be useful to add the explanation from the changelog as to why *Unlimited* is not recommended.

**Comment by dora-0** [comment]
I have added it. Thanks for your help!

File: wiki/Options/en.md [hunk 1]
```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
 | Name | Description |
 | --- | --- |
 | `VSync` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
-| `120fps` | Limits the game client to 120fps. |
-| `240fps` | Limits the game client to 240fps. |
+| `Power Saving` | Limits the game client to 2x refresh rate. |
+| `Optimal` | Limits the game client between 480~960fps based on 8x your monitor refresh rate. |
```
Comment (MegaApplePi) [inline wiki/Options/en.md]:
```suggestion
| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |
```

File: wiki/Options/en.md [hunk 2]
```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
 | Name | Description |
 | --- | --- |
 | `VSync` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
-| `120fps` | Limits the game client to 120fps. |
-| `240fps` | Limits the game client to 240fps. |
+| `Power Saving` | Limits the game client to 2x refresh rate. |
```
Comment (MegaApplePi) [inline wiki/Options/en.md]:
```suggestion
| `Power Saving` | Limits the game client to twice the refresh rate of your monitor. |
```

File: wiki/Options/en.md [hunk 3]
```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
...
+| `Optimal` | Limits the game client between 480~960fps based on 8x your monitor refresh rate. |
 | `Unlimited (gameplay)` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
 
 - `VSync` explanation: In layman terms, using VSync will force the game wait for the entire frame to load before displaying it.
   - You may want to use this if you see some "tearing" (when the bottom portion of the game is lagging behind the upper portion of the game).
   - Despite what the point above stated, this may cause some lag or slowdowns because the game client has to wait for each frame to load.
 - `Unlimited (gameplay)` explanation: Unlimited only applies to when you are playing a beatmap.
   - When you are doing anything but playing a beatmap, `240fps` is used instead.
```
Comment (MegaApplePi) [inline wiki/Options/en.md]:
```suggestion
  - When you are doing anything but playing a beatmap, the frame rate is limited to 240fps.
```

**Comment by dora-0** [comment]
Thank you for helping me!

File: wiki/Options/en.md [hunk 4]
```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
...
+| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |
 | `Unlimited (gameplay)` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
 
 - `VSync` explanation: In layman terms, using VSync will force the game wait for the entire frame to load before displaying it.
   - You may want to use this if you see some "tearing" (when the bottom portion of the game is lagging behind the upper portion of the game).
   - Despite what the point above stated, this may cause some lag or slowdowns because the game client has to wait for each frame to load.
 - `Unlimited (gameplay)` explanation: Unlimited only applies to when you are playing a beatmap.
-  - When you are doing anything but playing a beatmap, `240fps` is used instead.
+  - When you are doing anything but playing a beatmap, the frame rate is limited to 240fps.
+  - This option is **not recommended**. Using unlimited frame limiter can lead to stutters.
```
Comment (MegaApplePi) [inline wiki/Options/en.md]:
```suggestion
  - This option is **not recommended**. Using the `Unlimited (gameplay)` option can lead to stutters.
```

File: wiki/Options/en.md [hunk 5]
```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
...
+  - By using the `Optimal` setting, you will be automatically set to a frame limiter which (if your hardware can reach) offers imperceivable changes in system (input to output) latency when compared to `unlimited`.
```
Comment (MegaApplePi) [inline wiki/Options/en.md]:
```suggestion
  - Using the `Optimal` option instead offers imperceivable changes in system (input to output) latency when compared to `Unlimited (gameplay)`.
```

I think we can shorten this up to what I suggested. The `Optimal` setting was explained in the table above.

**Comment by cl8n** [comment]
some small changes to this will come in the next update

also I want to see if user config file changed

File: wiki/Options/en.md [hunk 6]
```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
...
+| `Power Saving` | Limits the game client to twice the refresh rate of your monitor. |
+| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |
 | `Unlimited (gameplay)` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
 
 - `VSync` explanation: In layman terms, using VSync will force the game wait for the entire frame to load before displaying it.
   - You may want to use this if you see some "tearing" (when the bottom portion of the game is lagging behind the upper portion of the game).
   - Despite what the point above stated, this may cause some lag or slowdowns because the game client has to wait for each frame to load.
 - `Unlimited (gameplay)` explanation: Unlimited only applies to when you are playing a beatmap.
-  - When you are doing anything but playing a beatmap, `240fps` is used instead.
+  - When you are doing anything but playing a beatmap, the frame rate is limited to 240fps.
```
Comment (cl8n) [inline wiki/Options/en.md]: 2x monitor refresh rate now

File: wiki/Options/en.md [hunk 7]
```diff
@@ -106,15 +106,17 @@ If you open the dropdown list for `Frame Limiter`, you will be presented with th
 | Name | Description |
 | --- | --- |
 | `VSync` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |
-| `120fps` | Limits the game client to 120fps. |
-| `240fps` | Limits the game client to 240fps. |
+| `Power Saving` | Limits the game client to twice the refresh rate of your monitor. |
+| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |
```
Comment (cl8n) [inline wiki/Options/en.md]:
```suggestion
| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, capping at 960fps. |
```

File: wiki/Options/en.md (hunk 6 above)
Comment (dora-0) [reply wiki/Options/en.md]: It seems to be still limiting to the 240fps, is it a 4x refresh rate? (cuttingedge 20191207.1)

Comment (cl8n) [reply wiki/Options/en.md]: 2x but lowest possible is 240, i guess

Comment (cl8n) [reply wiki/Options/en.md]:
```suggestion
  - When you are not playing a beatmap, the frame rate is limited to twice the refresh rate of your monitor, or 240fps, whichever is higher.
```
